o-markitdigital-equity-highlight/main.scss

```diff
@@ -30,71 +40,95 @@ $o-card-is-silent: false;
             font-weight: oFontsWeight(normal);
         }
 
-        &--bordered {
-            position: relative;
-            border: 1px solid #d3c5b4;
-            padding: 15px;
+        &__border {
+            border-bottom: 1px solid #cec6b9;
         }
 
         &__content {
             padding: 10px 0 0;
         }
 
         &__header {
-            font: 500 19px FinancierDisplayWeb;
-            color: #333333;
-            background-color: #fff1e0;
-            display: inline-block;
-            position: absolute;
-            left: 20px;
-            top: -10px;
-            padding: 0 5px;
+            font-size: 18px;
+            line-height: 20px;
+            font-family: MetricWeb,sans-serif;
+            margin: 0 0 20px;
+            font-weight: 700;
+
+            &:after {
+                content: "";
+                display: block;
+                width: 60px;
+                margin-top: 5px;
+                border-bottom: 4px solid #000000;
+                position: absolute;
+                z-index: 1;
+            }
         }
         
         &__footer {
             font-family: MetricWeb,sans-serif;
         }
 
         &__symbol {
-            width: 75%;
+            width: 60%;
             display: inline-block;
-            font: 600 18px/20px MetricWeb,sans-serif;
         }
 
         &__price {
             text-align: right;
-            width: 25%;
+            width: 40%;
             font: 600 18px/20px MetricWeb,sans-serif;
             float: right;
         }
 
-        &__price-change {
+        &__price-change,
+        &__price-change--1week {
+            width: 60%;
+            display: inline-block;
             margin-top: 8px;
             padding-top: 6px;
             border-top: 1px solid #e9decf;
```

Comment:
This should be a value from oColors (and the other colors which are currently hex values)

Reply:
done